packages/core/src/internalMonitoring.ts

```diff
@@ -14,7 +14,7 @@ export interface InternalMonitoring {
   setExternalContextProvider: (provider: () => utils.Context) => void
 }
 
-export interface MonitoringMessage {
+export interface MonitoringMessage extends utils.Context {
```

Comment:
At some point we might want to extract Context from `utils`. I doesn't make for a descriptive name.

Reply:
I am not sure to get your point, how extracting Context from utils would make the name more descriptive?